src/test/java/org/prebid/server/bidder/yieldlab/YieldlabBidderTest.java

```diff
@@ -301,6 +307,199 @@ public void makeBidsShouldReturnCorrectAdm() throws JsonProcessingException {
                 .containsExactly(expectedAdm);
     }
 
+    @Test
+    public void dsaIsForwarded() {
```

Comment:
please follow the code style of the test methods: `<method under test name>Should<expected result>When<given conditions>`